llvm/test/tools/sycl-post-link/device-code-split/per-aspect-split-4.ll

```diff
@@ -0,0 +1,55 @@
+; This test is intended to check that we do not perform per-aspect split if
+; it was disabled through one or another sycl-post-link option
+
+; RUN: sycl-post-link -symbols -S %s -o %t.table
+; RUN: FileCheck %s -input-file=%t.table --check-prefix CHECK-TABLE
+; RUN: FileCheck %s -input-file=%t_0.ll --check-prefix CHECK-IR
+;
+; -lower-esimd is needed so sycl-post-link does not complain about no actions
+; specified
+; RUN: sycl-post-link -lower-esimd -ir-output-only -S %s -o %t.ll
```

Comment:
should we also check -split=none?

Reply:
`-split=none` doesn't exists in `sycl-post-link`, it is represented as the lack of `-split` option.

Reply:
Ah yeah sorry